include/pagmo/population.hpp

```diff
@@ -411,6 +432,23 @@ class population
     }
 
 private:
+    // Short routine tp update the champion. Does nothing if the problem is MO
+    inline void update_champion(const vector_double& x, const vector_double& f)
+    {
```

Comment:
ok, but it looks like cheating, what happens if when invoking update_champion there is no memory to copy the arguments?